tests/test_gear_yaml2cfg.py

```diff
@@ -97,3 +112,19 @@ def test_yaml2cfg_test_no_header():
         shallow=False,
         )
     p.unlink()
+
+
+def test_yaml_duplicated_params(default_yaml_files):
```

Comment:
Isn't this being done in https://github.com/haddocking/haddock3/pull/913?

Reply:
I guess I messed up between PRs...